Visualisation/Node/Color/SecondaryStructureColorHeatmap.cs

```diff
@@ -0,0 +1,88 @@
+using System;
+using Nanover.Visualisation.Properties.Collections;
+using Nanover.Visualisation.Property;
+using UnityEngine;
+
+namespace Nanover.Visualisation.Node.Color
+{
+    /// <summary>
+    /// Colours protein residues using a heat-map according to some abstract metric.
+    ///
+    /// This will search the graph for a float array output node named "residue.normalised_metric_colour",
+    /// which stores a normalised metric value for each residue in the protein. These metric values
+    /// are then used in conjunction with a colour gradient object to determine the colour of each
+    /// residue. Note, it is imperative that the order of elements within the normalised metric
+    /// array matches the order in which residues are stored in the graph. This is normally not an
+    /// issue as the cartoon graphs always render all residues and, so far, maintain residue order.
+    /// </summary>
```

Comment:
Is this summary accurate? I don't see any code for searching for a named node, but rather than the node would provided in inspector as `residueNormalisedMetricColor`? If that is a correct understanding, then probably this class is more generally just applying gradient evaluation to an array of normalised scalars to arrive at an array of colors? In that case this could be renamed to reflect that?

Reply:
Sure it is hooked in via the inspector the but thing that is hooked in will perform the search (well the hypervisor class will perform the search).